### ISSUE TITLE
add js based local search

### ISSUE THREAD

**Issue by unuseless** [genielabs/zuix-web-template#2]
A great addition would be local javascript based search like:

- http://lunrjs.com/
- https://github.com/nextapps-de/flexsearch
- https://github.com/Tipue/Tipue-Search

or similar.
This would allow a reader of the book to offline search in the content of the book.

**Comment by genemars** [comment]
Yes, the internal search ([elasticlunr](https://github.com/weixsong/elasticlunr.js)) and much more  is now available in the newer and full featured template https://github.com/zuixjs/zuix-web-starter that replaces this one.
